client/src/components/new-request/search.vue

```diff
@@ -140,7 +140,7 @@
         </v-tooltip>
       </v-col>
       <v-col cols="4" class="ml-n8">
-        <v-tooltip top content-class="top-tooltip" transition="fade-transition" open-delay="200">
+        <v-tooltip top content-class="top-tooltip" transition="fade-transition">
```

Comment:
Same as above. Was put in initially due to the close proximity to the Search button.